frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -931,30 +963,35 @@ function useWMGQueryRequestBody() {
         organism_ontology_term_id: selectedOrganismId,
         self_reported_ethnicity_ontology_term_ids: ethnicities,
         sex_ontology_term_ids: sexes,
-        tissue_ontology_term_ids,
+        ...(version === 1 && { tissue_ontology_term_ids }),
       },
       is_rollup: true, // this could be made toggleable by users in the future
     };
   }, [
-    selectedGenes,
-    selectedTissues,
-    selectedOrganismId,
     data,
-    organismGenesByName,
-    tissuesByName,
+    selectedOrganismId,
+    version,
+    selectedTissues,
+    selectedGenes,
+    compare,
     datasets,
     developmentStages,
     diseases,
     ethnicities,
     sexes,
-    compare,
+    organismGenesByName,
+    tissuesByName,
   ]);
 }
 
-function useWMGFiltersQueryRequestBody() {
+function useWMGFiltersQueryRequestBody(
+  version: 1 | 2 = 1
+): FiltersQuery | null {
   const { selectedTissues, selectedOrganismId, selectedFilters } =
     useContext(StateContext);
-  const { data } = usePrimaryFilterDimensions();
+  console.log("calling usePrimaryFilterDimensions with version", version);
```

Comment:
do we want to keep this or just debug code?

Reply:
```suggestion
```